Parent/Parent/Create Account/CreateAccountViewController.swift

```diff
@@ -165,6 +165,43 @@ class CreateAccountViewController: UIViewController, ErrorViewController {
     func keyboardDidChangeState(keyboardFrame: CGRect) {
         scrollView.scrollToView(view: selectedTextField, keyboardRect: keyboardFrame)
     }
+
+    func termsOfServicePrivacyPolicyAttributedString() -> NSAttributedString {
+        let privacyPolicy = NSLocalizedString("Privacy Policy", comment: "Link text in 'By tapping ‘Create Account’, you agree to the %@ and acknowledge the %@.'")
+        let terms = NSLocalizedString("Terms of Service", comment: "Link text in 'By tapping ‘Create Account’, you agree to the %@ and acknowledge the %@.'")
+        let str = NSLocalizedString("By tapping ‘Create Account’, you agree to the %@ and acknowledge the %@.", comment: "")
+        let message = String.localizedStringWithFormat(str, terms, privacyPolicy)
+        let attributes = [
+            NSAttributedString.Key.font: UIFont.scaledNamedFont(.regular14),
+            NSAttributedString.Key.foregroundColor: UIColor.named(.textDark),
+        ]
+        let attributed = NSMutableAttributedString(string: message, attributes: attributes)
+        attributed.addAttribute(.foregroundColor, value: UIColor.named(.electric), range: (message as NSString).range(of: terms))
+        attributed.addAttribute(.foregroundColor, value: UIColor.named(.electric), range: (message as NSString).range(of: privacyPolicy))
+        attributed.addAttribute(
+            .link,
+            value: "https://www.instructure.com/policies/acceptable-use?newhome=canvas",
+            range: (message as NSString).range(of: terms)
+        )
+        attributed.addAttribute(
+            .link,
+            value: "https://www.instructure.com/policies/privacy?newhome=canvas",
+            range: (message as NSString).range(of: privacyPolicy)
+        )
+        return attributed
```

Comment:
It's not fun how much code this takes to do, but I think the result will be best for translators & end users. Strong work.

Reply:
Wow. Nice work. I'm surprised it's this complicated to get some tappable text in UIKit.